vault/core_util.go

```diff
@@ -55,6 +56,27 @@ func (c *Core) setupReplicationResolverHandler() error {
 	return nil
 }
 
+// GetCoreConfigInternal returns the server configuration
+// in struct format. This function should only be used internally
+// to retrieve the core config.
+func (c *Core) GetCoreConfigInternal() *server.Config {
```

Comment:
I added this initially because I was parsing listeners from core. I removed that functionality from Diagnose, but I think this helper method should still go in for later use.

Reply:
Does this and the next function repeatedly load the config?  Or is rawConfig set only once?

Reply:
Only once.

Reply:
Ok, np then.

Reply:
what do you mean by "should only be used internally"? to me that implies internal to the package, which would mean it might be better off as a private/unexported function?

Reply:
Yeah, that does seem a bit confusing. I just wanted to signify somehow that using the sanitized config is safer, but I'll remove that part of the comment.

Reply:
Removed! Resolving this conversation.